pkg/helm/helm.go

```diff
@@ -124,6 +124,11 @@ func AddHelmRepo(name, url string, update, helm3 bool) error {
 	if helm3 {
 		subdir = "helm3"
 	}
+
```

Comment:
Please add a comment here to explain what is happening? This didn't seem to be necessary for any of the other apps?

Reply:
sorry i missed updating this before the PR was merged,

Its done so we can use 1 generic name in the new LokiOptions -> Heml -> Repo -> Name section.
All the other apps pass in just the `char`t name, not `repo/chart` - this is backward compatible as other apps pass in the `chart` section only.